modules/cnf-configuring-workload-hints.adoc

```diff
@@ -26,3 +26,8 @@
 ----
 <1> If `highPowerConsumption` is `true`, the node is tuned for very low latency at the cost of increased power consumption.
 <2> Disables some debugging and monitoring features that can affect system latency.
+
+[NOTE]
+====
+When the `realTime` flag is set to `true` in a performance profile, include the `cpu-quota.crio.io: disable` annotation in the pod's YAML. Without this annotation the performance of the process within the pod is expected to be degraded.
```

Comment:
I would actually tell the user to add the annotation to every guaranteed pod with pinned cpus. 

Realtime workload hint (call it out explicitly to not confuse the user wrt realtime kernel) makes it worse, because it adds the nohz_full kernel argument.